### DISCUSSION TITLE
How to handle media play button and media loading indicator while media is loading.?

### DISCUSSION THREAD

**Discussion by kevalrathod** [muxinc/media-chrome#1010] [Q&A] [answered]
I am having having issue with media loading indicator and media player while media is playing.  
as you can see the screenshot while media is still in loading state the media play button  placement.  Is there any way to hide media play button while media is loading only. 

<img width="825" alt="image" src="https://github.com/user-attachments/assets/c73d184f-2013-4763-804a-df076fa47ad5">

one more, 
also when media is not loading media play button is not in the center possition while media is not loading. (second screenshot)

<img width="1470" alt="Screenshot 2024-10-16 at 4 00 17 PM" src="https://github.com/user-attachments/assets/6debc008-3b1a-4df0-8b1c-c4246f9be292">

**Comment by kevalrathod** [comment]
This is my css

```
media-loading-indicator:not([medialoading]) {
  display: none;
}

media-controller:not([mediahasplayed]) media-loading-indicator[medialoading][mediapaused] {
  display: none;
}

media-loading-indicator[medialoading] {
  display: block;
}

/* // hide center media button i media controller has played and media loading indicaror in loading  */
media-loading-indicator[medialoading] div[slot="centered-chrome"] media-play-button {
  display: none;
}
```

but it does not work. it still showung media button when loading

**Comment by luwes** [comment, marked as answer]
try this

```css
media-controller[medialoading]:not([mediapaused]) media-play-button {
  display: none;
}
```
